WordPressLint/src/main/java/org/wordpress/android/lint/WordPressIssueRegistry.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.lint
+
+import com.android.tools.lint.client.api.IssueRegistry
+import com.android.tools.lint.client.api.Vendor
+import com.android.tools.lint.detector.api.CURRENT_API
+import org.wordpress.android.lint.WordPressAndroidImportInViewModelDetector.Companion.ISSUE_ANDROID_IMPORT_IN_VIEWMODEL
+
+class WordPressIssueRegistry : IssueRegistry() {
```

Comment:
It isn't really crucial, but personally, I prefer the various sugars, like `override val`, etc. If you feel strongly about keeping it as a Java file, I'm not against that either.

Reply:
Thanks for the reply @mkevins , not feeling strongly about it at all, I actually prefer Kotlin as well, was just curious about it, that's all. Maybe if it was done as part of another separate `refactor` labeled commit, then this would have made it a bit more clearer to me that this was totally optional, a nice-to-have, many thanks! 👍